node/src/gql/query/index.ts

```diff
@@ -5,19 +5,21 @@ Permission to use, copy, modify, and distribute this software and its documentat
 The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
 */
 import { GraphQLObjectType } from 'graphql';
+import config from './config';
```

Comment:
I assume we'll also need an updateConfig mutation eventually so someone can edit the config settings in admin? I assume we'll want to limit that to admin users only, something to consider for the future.

Reply:
Yes. This just assumes the config is set, but we should have another story where User can edit config from /admin app